usaspending_api/database_scripts/matview_generator/summary_transaction_geo_view.json

```diff
@@ -190,30 +190,38 @@
       "name": "funding_subtier_agency_name",
       "where": "funding_subtier_agency_name IS NOT NULL",
       "columns": [{"name": "funding_subtier_agency_name"}]
+    }, {
+      "name": "simple_pop_geolocation",
```

Comment:
It looks like you're adding a couple new indices (`simple_pop_geolocation`, `simple_recipient_location_geolocation`) and I didn't see those listed in the technical description. It's pretty self explanatory but I just wanted to make a note for them.